src/utils/component-versioning.js

```diff
@@ -92,9 +92,13 @@ const defineVersioned = (dependencies, versionInfo, parentInstance) => {
     }
     // extract each dependant component's version,
     let { versions } = componentClass;
+    // If there is no version found, use the wrapping custom-element's version
     if (!versions && parentInstance) {
-      // taking the parent component's noted version for this dependency if needed,
+      // returns an array containing one object, which contains the tagname of the parent as key and its deployed version as value
       versions = versionInfo[parentInstance.constructor.tagName];
+      // extract the one item in the versions array and get only the version as string
+      // eslint-disable-next-line prefer-destructuring
+      versions[tagName] = Object.values(versions)[0];
```

Comment:
Did you test that with both builds? The `__VERSION_INFO__` object is different at these builds and i ran into a problem a time ago.

Reply:
What do you mean by that?